src/include/trn_datamodel.h

```diff
@@ -167,3 +167,8 @@ struct vsip_ppo_t {
 	__u8 proto;
 	__be16 port;
 } __attribute__((packed));
+
+struct ipv4_ct_tuple_t {
+	struct vpc_key_t vpc;
```

Comment:
it seems the only info we care here is tunnel_id. Do you think which one make code easier to read: the simple use of __u64 tunnel_id, or wrapping with vpc_key_t?